### PR TITLE
Add 10Cap valuation (closes #88)

### LINKED ISSUE #88: Add 10Cap valuation
Phil once mentioned [^1] he typically values each company using three methods: MOSP, Payback Time, and 10Cap.

If two methods roughly agree on a particular value, he uses it as his buy price.

[^1]: https://www.amazon.com/Invested-Warren-Buffett-Charlie-Emotions/dp/0062672649

Perhaps the 10Cap method would be useful for us here too, to compare against the other two.

If we decide to add it, that should not be excessively difficult.

The 10Cap formula is [^2]:

> Owners earnings = Net income + Non-cash charges - Maintenance capital expenditures (CapEx)

What we have, and what remains to be obtained:

1. Net income: OK (we have `net_income_history`),
2. Non-cash charges (how to fetch it?),
3. CapEx (I see it under MSN Money -> Financials).

[^2]: https://stablebread.com/how-to-calculate-and-analyze-warren-buffetts-owners-earnings/

### PR DESCRIPTION
**Pull request by kocielnik**
Example - MSFT above 10 Cap price:

msft-above-ten-cap-price

Example - PLBC below 10 Cap price:

plbc-below-ten-cap-price

### CODE REVIEW

**Comment by mrhappyasthma** [comment]
Let me know once this is ready for review :)

File: isthisstockgood/templates/loading.html [hunk 1]
```diff
@@ -90,7 +90,7 @@
 
             this.shadowRoot.append(style, wrapper);
 
-            this.animationTimeout = None;
```
Comment (kocielnik) [inline isthisstockgood/templates/loading.html:93]: Appeared in the browser console as:

cant-find-variable-none

File: isthisstockgood/templates/searchbox.html [hunk 2]
```diff
@@ -4,11 +4,11 @@
       <div class="col-md-3 nopadding">
         <input
             type="text"
+            autofocus
             class="form-control"
             id="ticker"
             placeholder="Ticker Symbol"
-            maxlength=8
-            onkeydown="return validateInput(event)"/>
```
Comment (kocielnik) [inline isthisstockgood/templates/searchbox.html:11]: Appeared in the browser console as:

cant-find-variable-validateInput

File: isthisstockgood/templates/searchbox.html [hunk 3]
```diff
@@ -4,11 +4,11 @@
       <div class="col-md-3 nopadding">
         <input
             type="text"
+            autofocus
```
Comment (kocielnik) [inline isthisstockgood/templates/searchbox.html:7]: Causes the search field to get activated automatically on each visit to the site (including page refresh).

@mrhappyasthma, if you see problems with this behaviour, please let me know! :)

**Comment by kocielnik** [comment]
> Let me know once this is ready for review :)

@mrhappyasthma, ready for review! :)

**Comment by kocielnik** [comment]
I've just noticed I needed to "undraft" it too. Done! :)

**Comment by kocielnik** [comment]
@mrhappyasthma, preview of the app with both pending PRs merged:

https://is-this-stock-good.vercel.app

**Comment by mrhappyasthma** [comment]
Looks good to me. The small fixes are all good as well. mostly obsolete stuff that we never removed.

**Pull request merged**